docs/source/api.rst

```diff
@@ -323,8 +360,9 @@ pcdsdevices.interface
 
     pcdsdevices.interface.BaseInterface
     pcdsdevices.interface.FltMvInterface
-    pcdsdevices.interface.LightpathInOutMixin
+    pcdsdevices.interface.LegacyLightpathMixin
```

Comment:
I guess we haven't regenerated these in a while

Reply:
Yeah, it generates during the docs `make` and we probably don't do it manually very often

Reply:
It's still generated each time in the actions, though, so this is only relevant for the master branch recording of `api.rst`:

(A link from last week)
https://github.com/pcdshub/pcdsdevices/blob/f7d38c582a48086cbdb0813d98504b66d2c7e0f7/master/_sources/api.rst.txt?plain=1#L363